### PR TITLE
Have "local ephemeral SSD" point to the Temporary Disk documentation.

### PR DESCRIPTION
**Pull request by treytro**
To clarify tempdb placement, I suggest we link "local ephemeral SSD" to point to the documentation for Temporary Disk for an Azure VM (azure/virtual-machines/managed-disks-overview#temporary-disk) to avoid confusion with Ephemeral OS Disk, which is something different than a Temporary Disk (https://learn.microsoft.com/en-us/azure/virtual-machines/ephemeral-os-disks).

### CODE REVIEW

File: azure-sql/includes/virtual-machines-best-practices-storage.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ ms.topic: include
   - For the log drive plan for capacity and test performance versus cost while evaluating the [premium P30 - P80 disks](/azure/virtual-machines/disks-types#premium-ssds)
     - If submillisecond storage latency is required, use [Azure ultra disks](/azure/virtual-machines/disks-types#ultra-disks) for the transaction log.
     - For M-series virtual machine deployments consider [write accelerator](/azure/virtual-machines/how-to-enable-write-accelerator) over using Azure ultra disks.
-  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the local ephemeral SSD (default `D:\`) drive for most SQL Server workloads that aren't part of Failover Cluster Instance (FCI) after choosing the optimal VM size.
+  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the [local ephemeral SSD](azure/virtual-machines/managed-disks-overview#temporary-disk) (default `D:\`) drive for most SQL Server workloads that aren't part of Failover Cluster Instance (FCI) after choosing the optimal VM size.
```
Comment (Jak-MS) [inline azure-sql/includes/virtual-machines-best-practices-storage.md]:
```suggestion
  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the [local ephemeral SSD](/azure/virtual-machines/managed-disks-overview#temporary-disk) (default `D:\`) drive for most SQL Server workloads that aren't part of Failover Cluster Instance (FCI) after choosing the optimal VM size.
```

**Comment by Jak-MS** [comment]
@MashaMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by bluefooted — APPROVED** [review]

**Comment by bluefooted** [comment]
#sign-off

**Comment by Court72** [comment]
@bluefooted 

PRMerger will accept sign-off from the listed author only. However, if you are a Microsoft employee authorized to update this content, you can send a request to [techdocprs@microsoft.com](mailto:techdocprs@microsoft.com) for this PR to be merged. Include your role and an explanation of why the PR should be merged without approval from the listed author.

**Comment by MashaMSFT** [comment]
#sign-off

**Comment by MashaMSFT** [comment]
#hold-off

File: azure-sql/includes/virtual-machines-best-practices-storage.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ ms.topic: include
   - For the log drive plan for capacity and test performance versus cost while evaluating the [premium P30 - P80 disks](/azure/virtual-machines/disks-types#premium-ssds)
     - If submillisecond storage latency is required, use [Azure ultra disks](/azure/virtual-machines/disks-types#ultra-disks) for the transaction log.
     - For M-series virtual machine deployments consider [write accelerator](/azure/virtual-machines/how-to-enable-write-accelerator) over using Azure ultra disks.
-  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the local ephemeral SSD (default `D:\`) drive for most SQL Server workloads that aren't part of Failover Cluster Instance (FCI) after choosing the optimal VM size.
+  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the [local ephemeral SSD](/azure/virtual-machines/managed-disks-overview#temporary-disk) (default `D:\`) drive for most SQL Server workloads that aren't part of Failover Cluster Instance (FCI) after choosing the optimal VM size.
```
Comment (MashaMSFT) [inline azure-sql/includes/virtual-machines-best-practices-storage.md]:
```suggestion
  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the temporary disk (the temporary disk is ephemeral, and defaults to D:\) for most SQL Server workloads that aren’t part of Failover Cluster Instance (FCI) after choosing the optimal VM size.
```

File: azure-sql/includes/virtual-machines-best-practices-storage.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ ms.topic: include
   - For the log drive plan for capacity and test performance versus cost while evaluating the [premium P30 - P80 disks](/azure/virtual-machines/disks-types#premium-ssds)
     - If submillisecond storage latency is required, use [Azure ultra disks](/azure/virtual-machines/disks-types#ultra-disks) for the transaction log.
     - For M-series virtual machine deployments consider [write accelerator](/azure/virtual-machines/how-to-enable-write-accelerator) over using Azure ultra disks.
-  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the local ephemeral SSD (default `D:\`) drive for most SQL Server workloads that aren't part of Failover Cluster Instance (FCI) after choosing the optimal VM size.
+  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the temporary disk (the temporary disk is ephemeral, and defaults to D:\) for most SQL Server workloads that aren’t part of Failover Cluster Instance (FCI) after choosing the optimal VM size.
```
Comment (MashaMSFT) [inline azure-sql/includes/virtual-machines-best-practices-storage.md]:
```suggestion
  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the temporary disk (the temporary disk is ephemeral, and defaults to D:\) for most SQL Server workloads that aren’t part of a failover cluster instance (FCI) after choosing the optimal VM size.
```

**Comment by MashaMSFT** [comment]
#sign-off

**Comment by rwestMSFT** [comment]
This PR has been replaced by internal pull request https://github.com/MicrosoftDocs/sql-docs-pr/pull/28106.

**Pull request closed without merging**